lib/XMLSerializer-impl.js

```diff
@@ -1,9 +1,12 @@
 "use strict";
 
-const { produceXMLSerialization } = require("./serialization");
+const serialize = require("../serialize.js");
```

Comment:
In no other jsdom related packages, `require` contains the file extension explicitly. Is there any reason for adding the file extension here?

Reply:
I believe we should move over to this style - partially because ES6 modules will require it anyways and partially to save stat calls in node at the current point in time.

Reply:
There is maybe an eslint rule that we can enable to make sure that all the `require` call includes the file extension.

Reply:
I think I should back out this change here. It is my preference (which is guided by the same things @Sebmaster mentions) creeping in to an unrelated change just because I touched those lines.

In general I think it would be a good project to have more uniform jsdom-org style rules, but we should do it systematically, and as you mention, it should be enforced by ESLint.